tools/darwin/packaging/ios/mkdeb-ios.sh.in

```diff
@@ -75,22 +81,26 @@ rm -rf $DIRNAME/$ARCHIVE
 mkdir -p $DIRNAME/$PACKAGE/DEBIAN
 if $ARM64; then
   echo "Package: $PACKAGE_ARM64"                  >  $DIRNAME/$PACKAGE/DEBIAN/control
-  echo "Name: @APP_NAME@-iOS (64-bit)"            >> $DIRNAME/$PACKAGE/DEBIAN/control
+  echo "Name: @APP_NAME@-${PP_DEVICE} (64-bit)"            >> $DIRNAME/$PACKAGE/DEBIAN/control
   echo "Pre-Depends: cy+cpu.arm64"                >> $DIRNAME/$PACKAGE/DEBIAN/control
   echo "Conflicts: $PACKAGE"                      >> $DIRNAME/$PACKAGE/DEBIAN/control
   echo "Replaces: $PACKAGE"                       >> $DIRNAME/$PACKAGE/DEBIAN/control
 else
   echo "Package: $PACKAGE"                        >  $DIRNAME/$PACKAGE/DEBIAN/control
-  echo "Name: @APP_NAME@-iOS"                     >> $DIRNAME/$PACKAGE/DEBIAN/control
+  echo "Name: @APP_NAME@-${PP_DEVICE}"                     >> $DIRNAME/$PACKAGE/DEBIAN/control
 fi
 echo "Priority: Extra"                            >> $DIRNAME/$PACKAGE/DEBIAN/control
-echo "Depends: firmware (>= 9.0)"                 >> $DIRNAME/$PACKAGE/DEBIAN/control
+if [ "@PLATFORM@" == "appletvos" ]; then
+  echo "Depends: firmware (>= @CMAKE_XCODE_ATTRIBUTE_TVOS_DEPLOYMENT_TARGET@)" >> $DIRNAME/$PACKAGE/DEBIAN/
+else
+  echo "Depends: firmware (>= @CMAKE_XCODE_ATTRIBUTE_IPHONEOS_DEPLOYMENT_TARGET@)"                 >> $DIRNAME/$PACKAGE/DEBIAN/control
```

Comment:
split this (change from hard coded 9.0 to @CMAKE_XCODE_ATTRIBUTE_IPHONEOS_DEPLOYMENT_TARGET@)  into a seperate commit before this commit

Reply:
Do you mean, one commit for iOS changes and another one for tvOS changes?